docker_test.go

```diff
@@ -46,6 +49,62 @@ func init() {
 	}
 }
 
+func TestWithinContainerStage1(t *testing.T) {
+	if providerType != ProviderDocker {
```

Comment:
suggestion: add a todo as this distinction will be removed by the context work
```suggestion
	// TODO: remove this skip check when context rework is merged.        
	if providerType != ProviderDocker {
```